### DISCUSSION TITLE
Changing the default user model

### DISCUSSION THREAD

**Discussion by danielh-official** [orchidsoftware/platform#2156] [Q&A] [answered]
In my project, I want to use two user tables. One for my standard end-users, and another for administrative users. The problems is that Orchid uses the "users" table by default.

Is there any way I can configure it to use another model/table?

**Comment by tabuna** [comment, marked as answer]
If we are talking about login and authorization, then you can change this using this parameter in the configuration:

https://github.com/orchidsoftware/platform/blob/5d31f233b3260c2757463045b8f37803d3db7935/config/platform.php#L69

**Reply by danielh-official** [reply]
While we're on this topic, I've renamed many default tables to start w/ the prefix "admin_". This is breaking my role functionality because the "addRole" method assumes the role/user table to be named "role_users", but my table is named "admin_role_users". Anyway I can extend that?
